docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/cards/media--ncids-feature-card-mm-image.html.twig

```diff
@@ -0,0 +1,14 @@
+{# The proper image display style for thumbnail must be set for this #}
+{% if content.field_image_promotional and content.field_image_promotional[0] %}
+  {{content.field_image_promotional}}
+{% else %}
+  {# Placeholder Image #}
+  {% set sourceAttribs = create_attribute({
+    'media': '(min-width: 880px)'
+  })
+  %}
+  <picture class="nci-card__image">
+    <source {{ sourceAttribs }} srcset="{{ get_placeholder_image('ncids_featured_4x3') }}">
+    <img src="{{ get_placeholder_image('ncids_featured_16x9') }}" alt="No Image Placeholder">        
```

Comment:
hardcoded image alt?

Reply:
This is the image placeholder alt, so it should not be an issue.